composer.json

```diff
@@ -30,6 +30,7 @@
         "guzzlehttp/guzzle": "^7.0",
         "guzzlehttp/promises": "^2.0",
         "guzzlehttp/psr7": "^2.0",
+        "laravel/horizon": "^5.4",
```

Comment:
`5.4` is the lowest version of Horizon that can be installed alongside Nightwatch as it added PHP `^8.0` support. Nightwatch itself only support `^8.2`.